src/Controls/tests/TestCases/MauiProgram.cs

```diff
@@ -53,6 +53,11 @@ protected override void OnAppLinkRequestReceived(Uri uri)
 		protected override Window CreateWindow(IActivationState activationState)
 		{
 			var window = base.CreateWindow(activationState);
+			
+			window.Created += (s, e) =>
+			{
+				Issues.Issue17865.Instance?.RevealLastItem();
```

Comment:
Can we move this event check into the Issue test itself? 

It'd be nice if we didn't have to litter `MauiProgram` up too much